nyc.config.js

```diff
@@ -6,5 +6,10 @@ const removeRootDirectory = pathArray => {
 
 module.exports = {
   include: [...removeRootDirectory(config.collectCoverageFrom), 'pages/**/*.js'],
-  exclude: [...removeRootDirectory(config.coveragePathIgnorePatterns), 'pages/api/__coverage__.js'],
+  exclude: [
+    ...removeRootDirectory(config.coveragePathIgnorePatterns),
+    'pages/api/__coverage__.js',
+    'common/**/index.js',
+    'components/**/index.js',
```

Comment:
Why not use `config.collectCoverageFrom` in some way?

Reply:
Sure.

I noticed that just removing ``<rootDir>`` was not working in this case since ``collectCoverageFrom`` is an array of regexp pattern strings whereas ``exclude`` is an array of glob patterns. 

I suppose I can use a function to turn strings form ``collectCoverageFrom`` into glob patterns.

Reply:
That's ideal. Definitely want to get away from defining these strings twice.